client/index.js

```diff
@@ -10,13 +10,15 @@
     document.querySelector('#messages').appendChild(logEntry);
   };
 
-  const checkUnlocked = () =>
-    new Promise((resolve, reject) => {
+  const checkUnlocked = async () => {
+    await window.ethereum.enable() // Ensure access to MetaMask
+    return new Promise((resolve, reject) => {
       web3.eth.getAccounts((err, accounts) => {
```

Comment:
Don’t we also have to wrap the web3 object like they do in the article you linked?

Reply:
This isn't required. They have opted not to remove the global web3 object. I tried to go for the most minimal change.

Reply:
Good to know, thanks!